cloud-sql/postgres/pgvector/notebooks/pgvector_gen_ai_demo.ipynb

```diff
@@ -0,0 +1,1149 @@
A6zoK/TltwKNaMi/EWuKJaGXQ61CyrQ/MaRjSurEe5COXxFvfpE9f4HT16VF7jJq50i46O9lhIoppWbGysFIls1/EVNrmQnEQFKSFuiXHiKj0lLT4+HufPn5fX+4lqVJ5qIp6YmBhZKaty5cqyepfSJqpcnTlzRlbHEpLVmjVrZGUu8bEQuObMmZNvpSyl66ixPytceT9rFK68z5wrkgAJkAAJaIkAhSstZZt7JQESIAFlBChcKePF3iRAAiRAAqomIH7o039hCsR/bW3uoyFoWrWQX5dX9a49HzyFK88z9faMFK68TZzrkQAJkID3CFC48h5rX1jJl4UrwcdsAZLSrbicaJVSVPbPw5XwM+itCAvSo2yEDtXL6FEiWAdxq1tsghW7jptx+LwFlxItyDArvz5QD6BitAFt6+jRoKIBFaJ0MIgX2UhAIYE///wTI0aMwKVLl+T1jq+//jr69OmjcBb1d6dw5f0cUrjyPnOuSAIkQAIkoCUCFK60lG3ulQRIgASUEaBwpYwXe5MACZAACaiawModGVi5MyPHHtrXM2Jqn2BV78vbwVO48jZxz69H4crzTDkjCZAACfgKAQpXvpIJ78Th68KVjYIQr1IzrYi5asGpyxZk/HfrnSJQQoQKCQRKlzDgYqIFu4+bEZdkhbhO0JUWHapD+3oGtKhuROkSQJBRfVWtXNk3xxQdgStXrkjpSlQBW7RoEUJDQ4tuMR+dmcKV9xND4cr7zLkiCZAACZCAlghQuNJStrlXEiABElBGgMKVMl7sTQIkQAIkoGoC+QlXTasZMLd/iKr35e3gKVx5m7jn16Nw5XmmnJEE1ELg2pVL+P2n79DjvofVEjI2f/YhWnfqhpJlyqkm5uIMlMJVcdL3/tpqEa5sZERVqpQMK45etOLcVTNMrtz9ByDTBBy5YMW5eLNL0AONOrSqaUT3xgZZNYsVrVzCyEEOCJhMJogrB5VeoegvQClceT+TFK68z5wrkgAJkAAJaIkAhSstZZt7JQESIAFlBChcKePF3iRAAiRAAqom8PNRM15em5pjDwM6BmJAp0BV78vbwVO48jZxz6+ndeFq24Y1SLh6GU1v6Yi6jZt5HrCCGS+dj8GOLRugMxjQZ+Bwp0e6Os7pBdjRLwnEnDiKZfOnISAoGJPmLFHNHl+bMAxpyckY/OxkVK1ZVzVxF1egFK6Ki3zxrOvLwpWQq8QffT7X84nX41OsOHbBgis3LDCZASU1qtIzgSOxVsQmOi9c6XRZFaxqldGhV/NA1CibfzWrDDNg0IESVvEcaa7qBwQoXHk/iRSuvM+cK5IACZAACWiJAIUrLWWbeyUBEiABZQQoXCnjxd4kQAIkQAKqJyCqXO2LMSMpzQpR3UoIV+HBvDpESWIpXCmh5Zt9tS5czZ8yBufPnETPBweh0+33FmuSjh7ciyVzp8BgMOK1peucjsXRuCsXz2Pt8oWoVK0m7nn4cafnK+qOJlMm1i1fCLPFgmYt26FRizaFLvnTti9x5uRRRERGoWe/wbL/jm++wNkzJ/H33t1o0OyWAudo27kHatZrlKePKSMDe3fvQMypEzh/5oT8CX/VWnVRrXZ91Kl/E8IiIu1jblxPwMaPl+WZ4/CfvyEjI12+3qxNpzzPm7Zqj8bNW+d5XeTtxJEDOHf6OK4nxqNCleqoUr0WGjdvi+jSZfPdj6hItXn96jzPDHo9SpergApVaqBC5WqFVn9KSbqBuS89A1NGOoaOn4aKVWvgr1934PD+PwrNha1Duy63o0bdhg77i/fV0YN/yb2FhoWj2z39HPaNj7uMvbt3IubkMVy5cB7lKlZB1Vp10Kx1J0RGl8ox7nLsOSycMQGBgUEYM+1NhISFOx2zFjtSuNJW1n1ZuLpy3YqryVaUi9ChRLAORkPe3IirBi8mWnH6igUJKc5fNahEuBLyVFiQDpVL6tG5gRHNquVjgAFIzRDxAruOmdGwsh4NK+bfT1snjLslAeUEKFwpZ+buCApX7hLkeBIgARIgARIoiACFK54PEiABEiABRwQoXPFskAAJkAAJkAAJKCRA4UohMB/sTuHKf4UrISlt+Ggp9Ho9pi36WAoqvtKEWCaEIyE3PT15VoFhWUwmTH1mEFKSk9CpRy+7cLV8/qs4vM85QeiBwSPRqmPXHOsIcWfVolm4eC4m3/VLREThyfGvoHylqvL51csX8NrzwxQj7HHfQ+ja60H7OCF5ffbhYvyx87t85woOCUXfwU+jScu2eZ6fO30Cb059rtAYbul4G3oPHA6DIR+jAcAHb83Aob9249ER49GkZTs536ZPlmHHlo2Fzm3r8OiwcWhyS/s8/YU8tfytGbhw9rT9WY06DTB84sx85z576jiWvfEKkm5cl891Op28ekm0yKiSGDLmJSmjZW97fv0Bn7w3X4psA59+wemYtdiRwpW2su7LwtXZaxbsjzHLqlKVS+mleBUZosu34lW6CYiNt+B8vBWJTohXzghXoqJViSAdqpXW4+bqBtxcXY+wwLy/aJGWCVxMsODAOTN+OWbB1SQLht8W5FDM0tYJ425JQDkBClfKmbk7gsKVuwQ5ngRIgARIgAQKIkDhiueDBEiABEjAEQEKVzwbJEACJEACJEACCglQuFIIzAe7U7jyX+Eq+cZ1bP96PSpWqYbm7br41Onbu+tHrF78hoxp4uz3HFZ0Es8P7/0dy9+cLvuOeWW+Xb6xCVelypZH7wEFi1DlKlWR8o6tCcnn3dcmycpU5StXRec7e6NazXpSUBJX7f2yfTNO/nMIYeEl8OTYqahYrSaEKHXy2N95OH636VPZV7Qnxk7N87x02fL2ilNCHntz2jjExpyC0RiAznfci5r1GyMyqpSstPbnrh9wZP+fco5eDz2Gjt3vyTFfduGq3+PPoERUtHxutVggKpqdO3NcVqoS0kXtBjdhyDMvwhiY86rc/bt/xqpFs2XFr2ETsriKdj3+Km5cTyzwnFy+eB4fvTtXVqyaPH
```

Comment:
nit: Let's remove these commented out lines

Reply:
Looks like because its a notebook the comment is showing up no on the proper lines:

Here are the three lines that can be removed: https://github.com/saketj/python-docs-samples/blob/main/cloud-sql/postgres/pgvector/notebooks/pgvector_gen_ai_demo.ipynb?short_path=92f0432#L343-L345

Reply:
Thanks for catching this, removed.